hash/page/page.go

```diff
@@ -64,7 +64,7 @@ func (d *Page) HasData() bool {
 }
 
 // New will create an instance
-func New(requestOffset int, pageSize int,
+func NewPage(requestOffset int, pageSize int,
```

Comment:
Why gxpage.New need change to NewPage?

Reply:
If u use New, u do not know what object u wanna to initialize.